src/Entity/ProductBundleOrderItemsAwareTrait.php

```diff
@@ -0,0 +1,30 @@
+<?php
```

Comment:
Why trait? Does that give us any value?

Reply:
Plugin author used trait to a similar case so I wanted to keep code consistency.

Reply:
I think it might be nice to convert https://github.com/BitBagCommerce/SyliusProductBundlePlugin/blob/master/src/Repository/ProductVariantRepository.php to trait
Extends - causes problems when We have multiple plugins that try to extend the same class
solution usually is to copy paste / merge new methods from both classes by hand to new class, with traits it is easier.

Reply:
Good point, we can refactor this easily.